apps/rush-lib/src/logic/InstallManager.ts

```diff
@@ -239,15 +242,24 @@ export class InstallManager {
             }
           }
 
-          return this._installCommonModules(shrinkwrapIsUpToDate,
-            options.allowShrinkwrapUpdates, options.collectLogFile)
+          return this._installCommonModules(shrinkwrapIsUpToDate, options)
             .then(() => {
+              const linkManager: BaseLinkManager = LinkManagerFactory.getLinkManager(this._rushConfiguration);
               if (!options.noLink) {
-                const linkManager: BaseLinkManager = LinkManagerFactory.getLinkManager(this._rushConfiguration);
                 return linkManager.createSymlinksForProjects(false);
               } else {
-                console.log(os.EOL
-                  + colors.yellow('Since "--no-link" was specified, you will need to run "rush link" manually.'));
+                // since our local links are out of date, go ahead and remove the rush-link.json
+                linkManager.deleteRushLinkJson();
+
+                if (options.skipInstall) {
```

Comment:
>skipInstall [](start = 28, length = 11)

In your testing you said that `--skip-install` is not actually faster.  Let's table this until [https://github.com/pnpm/pnpm/pull/1177](https://github.com/pnpm/pnpm/pull/1177) is sorted out.